pkg/relayer/proxy/relay_signer.go

```diff
@@ -0,0 +1,22 @@
+package proxy
```

Comment:
Can you leave a TODO in this file as well that the relayer proxy isn't the best place to have the signer.